.github/workflows/linting.yml

```diff
@@ -44,3 +46,5 @@ jobs:
         run: |
           python -m pip install --upgrade gitlint
           echo "$TITLE" | gitlint
+      - uses: stefanzweifel/git-auto-commit-action@v4
```

Comment:
Can you update this line to use the git hash the way we did with the other workflows?  (It was a recommendation as part of our OpenSSF scorecard efforts)

Example:
https://github.com/intel/cve-bin-tool/blob/main/.github/workflows/update-pre-commit.yml#L29
```yaml
      - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
```